internal/cli/daemon/daemon_test.go

```
@@ -0,0 +1,301 @@
+	if len(resp.Errors) > 0 {
+		t.Fatalf("graphql errors: %+v", resp.Errors)
+	}
+}
+
+// initRepo creates a minimal git repo with one commit so the git
+// provider can resolve HEAD. Returns the absolute repo path.
+func initRepo(t *testing.T, label string) string {
+	t.Helper()
+	dir := t.TempDir()
+	run := func(args ...string) {
+		cmd := exec.Command("git", args...)
```

golangci-lint (2.12.1)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)